### DISCUSSION TITLE
Compose key bindings with multiple modifiers

### DISCUSSION THREAD

**Discussion by romerosilva** [wezterm/wezterm#547] [Q&A]
Hi Wez,

first of all, great work! I've been using wezterm for a few days now as my main terminal and having a nice experience so far, apart from a few quirks here and there.

I have a few questions that I need some help with if you can:

1. How do I compose key bindings with multiple modifiers? For example, how do I achieve `Crtl + Shift + key`? I've tried a few combinations of multiple `mods` attribute, passing an array to `mods`, `mods="CRTL+SHIFT"` and others but no luck so far
2. Is there a way to close a tab without the prompt asking if I want to kill it?

**Comment by cxiao** [comment]
1. For combining modifiers, `mods="CTRL|SHIFT"` should work. It's at the bottom of this section of the docs: https://wezfurlong.org/wezterm/config/keys.html#configuring-key-assignments
2. See the docs for [`CloseCurrentTab`](https://wezfurlong.org/wezterm/config/lua/keyassignment/CloseCurrentTab.html?highlight=prompt#closecurrenttab), and the `confirm` option there. There is also a `confirm` option for  [`CloseCurrentPane`](https://wezfurlong.org/wezterm/config/lua/keyassignment/CloseCurrentPane.html?highlight=prompt#closecurrentpane) which you may also want!

**Comment by bew** [comment]
For 2, you can also set `window_close_confirmation = "NeverPrompt"`
See: https://wezfurlong.org/wezterm/config/lua/config/window_close_confirmation.html

**Comment by wez** [comment]
The other answers should help with your questions.  Regarding the quirks you mentioned, I'd love to hear about those to see what we can do about them!
